README.md

```diff
@@ -23,36 +23,120 @@ The program is currently in production, getting live feedback.
 ## Contributing to Ready Responder
 We have a Slack channel at [readyresponder.slack.com](https://readyresponder.slack.com) to give help if you need it.
 
-### Getting Started
+### Getting Started - Dependencies
 
-This is a Rails project that is configured to run on Ruby 2, and on a Postgres database.  So, the things you'll need to install before running ReadyResponder locally are Ruby, the `bundler` gem, and Postgres version 9.
+This is a Rails project that is configured to run on Ruby 2, and on a Postgres
+database. So, the things you'll need to install before running ReadyResponder locally are:
 
-1. Ruby: There's a detailed list of options for installing Ruby on the [official Ruby website](https://www.ruby-lang.org/en/documentation/installation/).
-  * [RVM](http://rvm.io/), [rbenv](https://github.com/rbenv/rbenv#readme), and [chruby](https://github.com/postmodern/chruby#readme) are common ruby installation managers for Macs & Linux.
-  * The exact version of Ruby that ReadyResponder is using is specified in the [.ruby-version](.ruby-version) file.
```

Comment:
It would be nice to keep this comment regarding .ruby-version. Some folks might not be aware of that convention.

Reply:
Agreed, added it back to the ruby setup section.